plugins/source/azure/resources/services/subscriptions/subscriptions.go

```diff
@@ -58,17 +53,27 @@ func Subscriptions() *schema.Table {
 				Type:     schema.TypeString,
 				Resolver: schema.PathResolver("State"),
 			},
+			{
+				Name:     "subscription_id",
```

Comment:
I'll probably revert this and use `ResolveAzureSubscription` with `SingleSubscriptionMultiplex`